testing/spectest/mainnet/capella/finality/BUILD.bazel

```
@@ -0,0 +1,14 @@
+load("@prysm//tools/go:def.bzl", "go_test")
+
+go_test(
+    name = "go_default_test",
+    size = "medium",
+    timeout = "short",
+    srcs = ["finality_test.go"],
+    data = glob(["*.yaml"]) + [
```

```suggestion
    data = [
```
Same here, there are no local yaml files that would match this glob